app/views/users/authorization_confirmation/new.html.erb

```diff
@@ -32,12 +32,12 @@
   </div>
 
   <div class='margin-top-4 center'>
-    <%= button_to(sign_up_completed_path, class: 'usa-button usa-button--big usa-button--wide') { t('user_authorization_confirmation.continue') } %>
+    <%= button_to(user_authorization_confirmation_path, class: 'usa-button usa-button--big usa-button--wide') { t('user_authorization_confirmation.continue') } %>
```

Comment:
Do we need `method: :post` here, since we're no longer routing to a `GET` path?

```suggestion
    <%= button_to(user_authorization_confirmation_path, class: 'usa-button usa-button--big usa-button--wide', method: :post) { t('user_authorization_confirmation.continue') } %>
```

Reply:
I tested and confirmed this button currently just reloads the page. Which now has me more concerned that the tests are showing as green 😅

Reply:
Hmm, `button_to` should default to a post. It was posting before

Reply:
Can you give me steps for reproducing the behavior where it reloads the page? I cannot figure out how to make it do that.

Reply:
I can't seem to reproduce it anymore. It may have been gremlins 🤷

Reply:
Looking at the code, and it should redirect you back to the SP in all cases. There may be a case where an SP sends you right back, but you wouldn't end up on this screen, you should end up on the redirect trap. Will keep an out for any weird behavior around this screen while working on completions.